contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -17,9 +18,14 @@
     RenderedTableContent,
 )
 from great_expectations.render.renderer.renderer import renderer
+from great_expectations.render.renderer_configuration import (
+    AddParamArgs,
+    RendererConfiguration,
+    RendererValueType,
+)
 
 
-class ExpectColumnValuesToBePresentInAnotherTable(QueryExpectation):
+class ExpectColumnValuesToBePresentInOtherTable(QueryExpectation):
```

Comment:
Changed to be consistent with file name

Reply:
Won’t this break the user’s workflow once we release V1? I’d vote to change the file name or add an alias

Reply:
The good news is that no one is using this Expectation yet. Would be happy to issue a PR into 1.0 so that everything is aligned.

Reply:
Agree with adding an alias. Would that just be a new parent class e.g. something like:
```
class ExpectColumnValuesToBePresentInAnotherTable(ExpectColumnValuesToBePresentInOtherTable):
    """Alias for previous name of this expectation."""
    pass
```

Reply:
Nevermind my suggestion above if it isn't released yet.